terraform/environments/analytical-platform-compute/iam-policies.tf

```diff
@@ -298,3 +298,50 @@ module "quicksight_vpc_connection_iam_policy" {
 
   tags = local.tags
 }
+
+data "aws_iam_policy_document" "data_production_mojap_derived_bucket_lake_formation_policy" {
+  #checkov:skip=CKV_TF_1:Module registry does not support commit hashes for versions
+  #checkov:skip=CKV_TF_2:Module registry does not support tags for versions
+  statement {
+    sid    = "AllowS3ReadWriteAPDataProdDerivedTables"
+    effect = "Allow"
+    actions = [
+      "s3:GetObject",
+      "s3:PutObject",
+    ]
+    resources = ["arn:aws:s3:::mojap-derived-tables/prod/*"]
+  }
+  statement {
+    sid    = "AllowS3AccessAPDataProdDerivedTablesBucket"
+    effect = "Allow"
+    actions = [
+      "s3:ListBucket",
+    ]
+    resources = ["arn:aws:s3:::mojap-derived-tables"]
+  }
+  statement {
+    sid    = "AllowLakeFormationCloudWatchLogs"
+    effect = "Allow"
+    actions = [
+      "logs:CreateLogStream",
+      "logs:CreateLogGroup",
+      "logs:PutLogEvents"
+    ]
+    resources = [
+      "arn:aws:logs:${data.aws_region.current.id}:${data.aws_caller_identity.current.account_id}:log-group:/aws-lakeformation-acceleration/*",
+      "arn:aws:logs:${data.aws_region.current.id}:${data.aws_caller_identity.current.account_id}:log-group:/aws-lakeformation-acceleration/*:log-stream:*"
+    ]
```

Comment:
Does this log group exist already or will something create it?

Reply:
I believe the lake formation service will create it. It's [required for roles that register locations](https://docs.aws.amazon.com/lake-formation/latest/dg/registration-role.html) (which this role will be used for)